deps/verifier/src/snp/mod.rs

```diff
@@ -307,6 +313,41 @@ fn get_common_name(cert: &x509::X509) -> Result<String> {
     Ok(e.data().as_utf8()?.to_string())
 }
 
+// Function to request vcek from KDS asynchronously. Return vcek in der format.
```

Comment:
We can change `//` to `///`. This would automatically [generate a document](https://doc.rust-lang.org/rust-by-example/meta/doc.html#doc-comments) about the function when we publish the crate to `crates.io`.

Reply:
Do you want me to add description to all functions in the file?